metadata-transformer/src/test/java/no/unit/nva/metadata/service/testdata/TypeInformationArgumentsProvider.java

```diff
@@ -0,0 +1,20 @@
+package no.unit.nva.metadata.service.testdata;
+
+import no.unit.nva.metadata.type.Citation;
+import no.unit.nva.model.contexttypes.Book;
+import no.unit.nva.model.contexttypes.Journal;
+import no.unit.nva.model.instancetypes.book.BookMonograph;
+import no.unit.nva.model.instancetypes.journal.JournalArticle;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+
+import java.util.stream.Stream;
+
+public class TypeInformationArgumentsProvider implements ArgumentsProvider {
+    @Override
+    public Stream<? extends Arguments> provideArguments(ExtensionContext context) throws Exception {
+        return Stream.of(Arguments.of(Citation.ISBN.getMetaTagName(), "9783110646610", Book.class, BookMonograph.class),
```

Comment:
the ISBN and the ISSN value should be static fields  with names VALID_BOOK_ISBN (although ISBN is only for books) and the ISSN should be VALID_ISSN_FOR_JOURNAL_ARTICLE.

Reply:
In general there is an implicit content on these numbers that should be made explicit.